### DISCUSSION TITLE
Is Plane Surface 32 in WP2_LOH1 the Physical Surface (100) by default?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#823] [Q&A]
I see that the Plane Surface 32 in WP2-LOH1(https://github.com/SeisSol/Examples/tree/master/WP2_LOH1) does not allocate Physical surface. Is it the default Physical Surface (100)?
![mmexport1679252388574](https://user-images.githubusercontent.com/104721358/226203669-b451a37a-a9e9-4303-9d02-e530b44df206.jpg)

**Comment by Thomas-Ulrich** [comment]
https://github.com/SeisSol/Examples/blob/master/WP2_LOH1/loh.geo#L65
101 and 105 in gmsh will be changed to 1 and 5 in the puml mesh format.

**Reply by smallriver666** [reply]
> https://github.com/SeisSol/Examples/blob/master/WP2_LOH1/loh.geo#L65 101 and 105 in gmsh will be changed to 1 and 5 in the puml mesh format.

Yes, I know this, but I want to ask Plane Surface 32, it is the common surface beteen Volumn 1 and Volume 2.

**Reply by Thomas-Ulrich** [reply]
yes, the default is regular.
